.pre-commit-config.yaml

```diff
@@ -60,7 +60,7 @@ repos:
             "--load-plugins=pylint.extensions.docparams",
           ]
         # disabled plugins: pylint.extensions.mccabe
-        exclude: tests/functional/|tests/input|tests/extensions/data|tests/regrtest_data/|tests/data/|doc/
+        exclude: tests/functional/|tests/input|tests/extensions/data|tests/regrtest_data/|tests/data/|tests/pyreverse/data/|doc/
```

Comment:
This is not the correct place for the exclude pattern. You can add different excludes for each hook, and my guess is that the HTML reformatting is performed by ``prettier`` (the last hook in the file).
If you install ``pre-commit`` on your own machine, you can see which hook changes which files before you commit.

You can install it by following the pylint docs:

> * install pre-commit using ``pip install pre-commit``
> * then run ``pre-commit install`` in the pylint root directory to enable the git hooks.

Reply:
thank you for your suggestions, i really appreciate those. I'm using gitpod to edit, and seems that pre-commit has an issue in that behavior. After a (not so) quick and a (really) dirty fix i used pre-commit in this environment.

I hope it's ok now.

Thank you for your support